apple/swift-docc-render
# Use builtin image as fallback when authored image fails to load.

Bug/issue #, if applicable: 83366845

## Summary

Updates the renderer to display the following image for the scenario where the image that an author has specified is not found or invalid for any reason.

![Fallback image](https://raw.githubusercontent.com/mportiz08/swift-docc-render/image-fallback/src/assets/img/no-image%402x.png)

## Testing

Steps:
1. Update an existing documentation catalog or source comment to include the following markdown:
```markdown
![blah](/fake/image/path.jpg)
```
2. Build this branch with `npm run build` and configure `DOCC_HTML_DIR=/path/to/swift-docc-render-dist` when previewing those docs with `docc preview`
3. Verify that the fallback image displays where the fake image markdown was added
4. Verify that real images still render as before

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
